src/Notes/MarketingNotifications.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Marketing notifications for merchants (admin).
+ *
+ * @package Pinterest_For_WooCommerce/Classes/
+ * @since   x.x.x
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Pinterest\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class responsible for displaying inbox notifications for the merchant.
+ *
+ * In the specification for the notes we have that the notification should
+ * be sent some time after the plugin installation. There is no retroactive
+ * way of figuring out when the plugin was first installed. So we count
+ *
+ * @since x.x.x
+ */
+class MarketingNotifications {
+
+	// All options common prefix.
+	const OPTIONS_PREFIX = PINTEREST_FOR_WOOCOMMERCE_PREFIX . '-marketing-notifications';
+
+	// Timestamp option marking the moment we start to count time.
+	const INIT_TIMESTAMP = self::OPTIONS_PREFIX . '-init-timestamp';
+
+	// List of marketing notifications that we want to send.
+	const NOTES = array(
+		'EnableCatalogSync',
+		'CatalogSyncErrors',
+		'CompleteOnboardingAfterThreeDays',
+		'CompleteOnboardingReminderAfterSevenDays',
+		'CompleteOnboardingReminderAfterFourteenDays',
+		'CompleteOnboardingReminderAfterThirtyDays',
+	);
+
+	/**
+	 * Trigger inbox messages.
+	 *
+	 * @since x.x.x
+	 * @return void
+	 */
+	public function init_notifications(): void {
+
+		foreach ( self::NOTES as $note ) {
+			$note         = 'Automattic\WooCommerce\Pinterest\Notes\Collection\\' . $note;
+			$notification = new $note();
```

Comment:
This change should be coupled with the explanation in the comment above regarding the use of `::class`. We can also type-hint against our `AbstractNote` class.

```suggestion
			/** @var AbstractNote $notification */
			$notification = new $note();
```